lightspeed/modules/con_view-manage-admin-dashboard-telemetry.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: CONCEPT
+
+[id="view-manage-admin-dashboard-telemetry_{context}"]
+
+= Viewing and managing the Admin dashboard telemetry
+
+
+{LightspeedshortName} collects the following telemetry data by default:
+
+* *Operational telemetry data*
++
+This is the data that is required to operate and troubleshoot the Ansible Lightspeed service. For more information, refer the Enterprise Agreement. You cannot disable the collection of operational telemetry data. 
++
+This includes the following data:
+
+** Organization you are logged into (Organization ID, account number)
+** Large language model (or models) that you are connected to 
+
+* *Admin dashboard telemetry data*
++
+This is the data that provides insight into how your organization users are using the Ansible Lightspeed service, and the metrics are displayed on the Admin dashboard. 
++
+This includes the following data:
+
+** Prompts and content suggestions, including accept or reject of the content suggestions
+** User sentiment feedback
++
+You can also disable the Admin dashboard telemetry if you no longer want to collect and monitor the telemetry data. 
+
+[NOTE]
+====
+Viewing telemetry data on the Admin dashboard is not yet supported on {LightspeedShortName} on-premise deployments.
+====
+
+== Prerequisites
+To view and manage the Admin dashboard telemetry data, ensure that you have the following:
+
+* You have organization administrator privileges to a Red Hat Customer Portal organization with a valid Red Hat Ansible Automation Platform subscription.
+
+* You have installed the Ansible VS Code extension v2.13.148 that is required to collect Admin dashboard telemetry. 
+
+IMPORTANT: {LightspeedshortName} does not collect users' personal information, such as usernames or passwords. If any personal information is inadvertently received, the data is deleted. For more information about {LightspeedshortName}'s privacy practices, see the xref:telemetry-data-collection-notice_lightspeed-intro[Telemetry Data Collection Notice for the Admin dashboard].
```

Comment:
Hi @sayjadha. I know you've gone through some iterations, but wanted to point out the formatting for the admonition here. Of course, it's up to you whether to incorporate that change here.
```suggestion
[IMPORTANT]
====
{LightspeedshortName} does not collect users' personal information, such as usernames or passwords. If any personal information is inadvertently received, the data is deleted. For more information about {LightspeedshortName}'s privacy practices, see the xref:telemetry-data-collection-notice_lightspeed-intro[Telemetry Data Collection Notice for the Admin dashboard].
====
```

Reply:
Incorporated your feedback, thank you for catching that!